cocotb/_build_libs.py

```diff
@@ -356,7 +374,7 @@ def get_ext():
         mti_path = os.path.join(modelsim_include_dir, "mti.h")
         if os.path.isfile(mti_path):
             fli_ext = Extension(
-                os.path.join("cocotb", "libs", "modelsim", "libfli"),
+                os.path.join("cocotb", "libs", "libfli_modelsim"),
```

Comment:
```suggestion
                os.path.join("cocotb", "libs", "libcocotbfli_modelsim"),
```
Don't really care either way. Shouldn't the word cocotb be in the lib name though?